### PR TITLE
Add Conference preferences

### PR DESCRIPTION
**Pull request by degouville**
## Description

- Offers the option to choose the default conference (East or West) and to view standings for the whole league. It helps to compare the results of teams in different conferences.

## Screencast

|Before|After|
| --- | --- |
|||

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/nba-game-viewer/src/hooks/useStandings.ts [hunk 1]
```diff
@@ -34,8 +34,18 @@ const useStandings = () => {
...
     const westernStandings = getConferenceStandings(westernConference);
+    const leagueStandings = getConferenceStandings({
+      name: "League",
+      abbreviation: "L",
+      standings: {
+        name: "League Standings",
+        entries: [...easternConference.standings.entries, ...westernConference.standings.entries],
+      },
+    }).sort((a: Team, b: Team) => {
+      return a?.wins !== b?.wins ? (b?.wins || 0) - (a?.wins || 0) : (a?.losses || 0) - (b?.losses || 0);
```
Comment (degouville) [inline extensions/nba-game-viewer/src/hooks/useStandings.ts:45]: e.seed give the ranking per Conf but to sort All of them, we use this algorythm:
- 1st we rank every teams by wins descreasingly
- if there's equalities, we rank concurrent by putting the ones with less losses first

**Comment by pernielsentikaer** [comment]
Thanks for the update @degouville, do you mind adding a [CHANGELOG](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) entry too?

 @AbhinavPalacharla do you mind checking this?

**Comment by pernielsentikaer** [comment]
What about @djpowers @michaelli1324 - do you see any issues with this?

**Comment by djpowers** [comment]
> What about @djpowers @michaelli1324 - do you see any issues with this?

Looks good after a quick check to me. One small thing, does the list title show up as "League Conference" as in the screenshot? I'd expect to see "Eastern Conference," "Western Conference," and then just "League" with that option selected.

**Comment by degouville** [comment]
Oh good catch. Doesn't makes sense.

**Comment by degouville** [comment]
And now we have an updated PR with a screenshot of the Celtics on top... (😏 I see what you did here @djpowers! 😌)

**Comment by pernielsentikaer** [comment]
Is this ready yet, or do you miss something @degouville

**Comment by degouville** [comment]
It is fixed here https://github.com/raycast/extensions/pull/9165/commits/0fe83f1d0dd7b51c8c4720931fc7f06984123c7b @pernielsentikaer. We are good to go now.

**Review by djpowers — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**